cmd/cosign/policy_webhook/main.go

```diff
@@ -33,9 +33,9 @@ import (
 	"knative.dev/pkg/webhook/resourcesemantics/validation"
 	"sigs.k8s.io/release-utils/version"
 
-	"github.com/sigstore/cosign/pkg/apis/cosigned"
-	"github.com/sigstore/cosign/pkg/apis/cosigned/v1alpha1"
-	"github.com/sigstore/cosign/pkg/apis/cosigned/v1beta1"
+	"github.com/sigstore/cosign/pkg/apis/policycontroller"
```

Comment:
How about `policy` instead of `policycontroller`? Unless the resources are letting you configure the controller (like an operator), the resources you're actually operating on are the policies, right?

Reply:
Sgtm